services/explorer/src/models/TableData.ts

```diff
@@ -42,6 +44,14 @@ export interface NodesTableData {
   pbftCount: number;
 }
 
+export interface HoldersTableData {
+  rank: number;
+  address: string;
+  balance: BigNumber;
+  totalSupply: BigNumber;
+  taraPrice: number;
+}
```

Comment:
If it has to do only with the Holders, we can move it to a specific file.

Reply:
NodesTableData, TransactionTableData and BlockData are all here in this file. Should we break them in an `interfaces.ts` file near the tables?

Reply:
Yeah. Let's do it